examples/js/nodes/materials/nodes/MeshStandardNode.js

```diff
@@ -48,6 +48,15 @@ MeshStandardNode.prototype.build = function ( builder ) {
 
 		this.color = map ? new OperatorNode( color, map, OperatorNode.MUL ) : color;
 
+		// slots
+		// * ao
+		// * aoMap
+
+		var ao = builder.findNode( props.ao, inputs.ao ),
```

Comment:
Oops this should be aoMapIntensity and included in shortcuts

Reply:
Yeah. To shortcuts in `MeshNodeMaterial` you can add here:
https://github.com/donmccurdy/three.js/blob/6a498fd2793befd33aa315716ae2af2022c71bb9/examples/js/nodes/materials/nodes/MeshStandardNode.js#L18
And
https://github.com/donmccurdy/three.js/blob/6a498fd2793befd33aa315716ae2af2022c71bb9/examples/js/nodes/materials/nodes/MeshStandardNode.js#L25

Reply:
https://github.com/donmccurdy/three.js/blob/6a498fd2793befd33aa315716ae2af2022c71bb9/examples/js/nodes/materials/MeshStandardNodeMaterial.js#L29